tests/ci/clean_up.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 set -euo pipefail
 
+# Get OS data.
+source /etc/os-release
+
 # Restore the *-release files
 cp -fv "${TMPCI_DIR}"/os-release /etc/
 cp -fv "${TMPCI_DIR}"/redhat-release /etc/
 
-source /tmp/.env
-
 ID=${ID:-}
-IMAGE_KEY=${IMAGE_KEY:-}
+ARCH=$(arch)
+UUID=${UUID:-local}
+IMAGE_KEY="auto-${ARCH}-${UUID}"
```

Comment:
You are repeating code for setting variables and sourcing the /etc/os-release file in each script, would it make sense to move the common parts to a helper lib ?

Reply:
I thought about it, it was just a couple of lines and I was trying to separate the scripts as much as possible, so they can run in different environments, maybe even in different directories to run inside different containers as Liora suggested.
So, right now I'm trying to make them as independent and autonomous as I can.

Reply:
That's fine something to think about later.